src/styles/Overview.scss

```diff
@@ -0,0 +1,45 @@
+@import "utils/colors";
+
+.Overview {
+  background-color: $WHITE;
+  border-radius: 8px;
+  box-shadow: 0 0 16px 0 rgba(17, 17, 26, 0.05);
+  width: 880px;
```

Comment:
i believe you mentioned this, but there are some responsive breakpoints setup that should get added throughout so we support tablet, but esp mobile device viewing.